.idea/runConfigurations/Debug_Elasticsearch__node_2_.xml

```diff
@@ -6,6 +6,10 @@
     <option name="HOST" value="localhost" />
     <option name="PORT" value="5008" />
     <option name="AUTO_RESTART" value="true" />
+    <RunnerSettings RunnerId="Debug">
+      <option name="DEBUG_PORT" value="5008" />
+      <option name="LOCAL" value="false" />
+    </RunnerSettings>
```

Comment:
not sure why intellij keeps changing this file... will make sure to remove this before merging